vtex-apps/app-store
# Move local storage setting to confirm button

#### What is the purpose of this pull request?

Move `localStorage` setting to the `ConfirmButton` handler.

#### What problem is this solving?

With new Apps Admin rollout, the App Store redirect feature from the Admin has been deprecated. Thus, the `Header` feature that transpose the query string value to the local storage is not needed anymore.

#### How should this be manually tested?

1. Visit this [linked Review page](https://accountreview--extensions.myvtex.com/social-selling/r)
2. Click in the `Confirm Order` button
3. Go back to the linked App Store. Now, the account value you used persists in any account input until changed

#### Screenshots or example usage
None

#### Types of changes
